### PR TITLE
Task/new table view/723

### PR DESCRIPTION
**Pull request by Kyubinhan**
Change table titles and use Accordian component

Test: https://wps-pr-742.pathfinder.gov.bc.ca/morecast

### CODE REVIEW

**Review by andrea-williams — APPROVED** [review]
You might want to check in with Ambrose on these changes (specifically that the tables are now collapsed by default) before merging into prod, since it might affect his usability testing next week (?)

File: web/src/features/fireWeather/components/WxDataDisplays.tsx
Comment (Sybrand) [inline web/src/features/fireWeather/components/WxDataDisplays.tsx:107]: As per discussion - please make sure this wording is OK with UX and with PO.

**Review by Sybrand — APPROVED** [review]
Please check with UX and P.O. re. text changes, and starting collapsed. If they're OK with it - then this has my approval.

**Review by ambroseyau — APPROVED** [review]

**Pull request merged**
